### DISCUSSION TITLE
Livewire

### DISCUSSION THREAD

**Discussion by muhammadhanief** [icehouse-ventures/laravel-chartjs#6] [Q&A]
Can this work in Livewire? Because i want to create real time chart with dynamic parameter.

**Comment by peterjthomson** [comment]
We're working on a Livewire implementation of the package. But for now, there are a couple of options you can try out. There's a Livewire Chartjs package here: https://github.com/asantibanez/livewire-charts Also, Laravel Pulse uses Livewire and Chartjs, there's an example of a live updating chart here: https://github.com/laravel/pulse/blob/1.x/resources/views/livewire/queues.blade.php
